src/components/pages/JobDetails.vue

```diff
@@ -128,13 +128,15 @@ export default {
                    itemscope
                    itemtype="http://schema.org/Organization"
               >
-                <img
-                  itemprop="logo"
-                  :src="postData.company.logo"
-                  :alt="postData.company.name"
-                  ref="logo"
-                  @error="handleImageError"
-                >
+                <a :href="postData.company.www" target="_blank">
```

Comment:
Nope, as explained in the picture in the Issue;
When the company logo is clicked, it should go to the 'CompanyJobListing' page.

`<router-link>` must be used.

Reply:
Thanks for the feedback, I totally misunderstood the issue.